website/docs/configuration.md

```diff
@@ -76,6 +77,7 @@ as such it's also documented in `:h guifont`. But to sum it up and also add Neov
     should Hack fail to contain any glyph.
   - `Roboto_Mono_Light:h10` — Roboto Mono Light at size 10.
   - `Hack:h14:i:#e-subpixelantialias:#h-none`
+  - `Iosevka+ss10,Symbols_Nerd_Font:h22` — Iosevka at size 22, and enable stylistic sets `ss10`.
 
```

Comment:
I was first going to suggest that you move the features to options, which are already parsed, but then it occurred to me that, the features are font-specific, and should be applied per font.

And after even more reflection, I think most of the options should actually be font specific, rather than applying to the whole list. I think at some point we need to come up with another more flexible format, and deprecate the current one, but I don't know how. Anyway, that should probably not be done here in this PR.

Reply:
Maybe we need a table instead of the simple string? [Font settings in wezterm](https://wezfurlong.org/wezterm/config/font-shaping.html#advanced-font-shaping-options) seems simple and clear.

Reply:
Yes, I think something like that makes sense. Maybe we could use the guifont to specify the name and fallbacks, and the size. And then a lua table with the font names for the options.

I’m leaving this open for a bit still, so that we have time to think about it.